tests/plugins/webp-uploads/helper-tests.php

```diff
@@ -215,6 +215,10 @@ public function it_should_prevent_to_create_an_image_size_when_attached_file_doe
 	 * @test
 	 */
 	public function it_should_prevent_to_create_a_subsize_if_the_image_editor_does_not_exists(): void {
+		if ( ! wp_image_editor_supports( array( 'mime_type' => 'image/webp' ) ) ) {
+			$this->markTestSkipped( 'Mime type image/webp is not supported.' );
+		}
```

Comment:
For some reason this test is [failing](https://github.com/WordPress/performance/actions/runs/9022524551/job/24792443467?pr=1205) in PHP 7.2 now with:

```
There was 1 failure:

1) WebP_Uploads_Helper_Tests::it_should_create_an_image_with_filter_webp_uploads_pre_generate_additional_image_source
Failed asserting that WP_Error Object &0000000004c81e5c000000005c809afd (
    'errors' => Array &0 (
        'image_mime_type_not_supported' => Array &1 (
            0 => 'The provided mime type is not supported.'
        )
    )
    'error_data' => Array &2 ()
    'additional_data' => Array &3 ()
) is of type "array".

/var/www/html/wp-content/plugins/performance/tests/plugins/webp-uploads/helper-tests.php:294
```

So I added this.

Reply:
Ah, it's because of the additional validation I added. Fixed in https://github.com/WordPress/performance/pull/1205/commits/d8f886b24f298eaaf113bc7c4f78b33296707889.